client/paperflow-web-v1/src/pages/KeywordNetwork/index.js

```diff
@@ -25,21 +43,59 @@ const App = () => {
           });
         select(svgRef.current).call(container);
       }
-    }, 1000);
+    }, 10);
     return () => clearTimeout(timer);
   }, []);
 
+  const [networkData, setNetworkData] = useState({
+    nodes: [],
+    links: [],
+  });
+
+  let i;
+  let j;
+  useEffect(() => {
+    const newData = {
+      nodes: [],
+      links: [],
+    };
+    for (i = 0; i < data.nodes.length; i += 1) {
+      for (j = 0; j < data.nodes[i].length; j += 1) {
```

Comment:
use `data.nodes.map( (item) => {} )` instead of `for` loops as possible.

same issue for the following lines